actionmailer/lib/action_mailer/delivery_methods.rb

```diff
@@ -12,7 +12,7 @@ module DeliveryMethods
       # Do not make this inheritable, because we always want it to propagate
       cattr_accessor :raise_delivery_errors, default: true
       cattr_accessor :perform_deliveries, default: true
-      cattr_accessor :deliver_later_queue_name, default: :mailers
+      cattr_accessor :deliver_later_queue_name, default: :default
```

Comment:
We can't change this value. This will change the default value for all the applications that don't want to opt-in for the Rails 6.1 defaults.

Reply:
@rafaelfranca when we use `deliver_later_queue_name` then delivery jobs got `mailers` queue instead of `default`. So I have to change the default configuration from `nil` to `:default`

Reply:
I'm checking other options to set `default` for `deliver_later_queue_name = nil`

Reply:
my bad, removed

Reply:
My bad, removed it